proxy/src/control/pb.rs

```diff
@@ -45,10 +45,16 @@ impl event::StreamResponseEnd {
             eos,
         };
 
+        let destination_meta = ctx.dst_labels()
+            .and_then(|b| b.borrow().cloned())
+            .map(|d| tap_event::EndpointMeta {
+                labels: d.as_map().clone(),
+            });
+
         common::TapEvent {
             source: Some((&ctx.server.remote).into()),
             destination: Some((&ctx.client.remote).into()),
-            destination_meta: None,
+            destination_meta,
```

Comment:
i don't really know rust, but is an assignment or type needed here? just noting `destination_meta` is not like the others.

Reply:
When there's a variable in scope with the same name and type as a struct field, it can be initialized to that value with this syntax.